packages/compass/src/main/menu.ts

```diff
@@ -55,36 +82,36 @@ function networkOptInDialogItem() {
   };
 }
 
-function darwinCompassSubMenu() {
+function darwinCompassSubMenu(): MenuItemConstructorOptions {
   return {
     label: app.getName(),
     submenu: [
       {
         label: `About ${app.getName()}`,
-        selector: 'orderFrontStandardAboutPanel:',
+        role: 'about',
       },
       separator(),
       {
         label: 'Hide',
         accelerator: 'Command+H',
-        selector: 'hide:',
+        role: 'hide',
       },
       {
         label: 'Hide Others',
         accelerator: 'Command+Shift+H',
-        selector: 'hideOtherApplications:',
+        role: 'hideOthers',
       },
       {
         label: 'Show All',
-        selector: 'unhideAllApplications:',
+        role: 'unhide',
```

Comment:
Might want to move these fixes into a separate pr with a ticket attached
https://www.electronjs.org/docs/latest/api/menu-item#roles